src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -118,14 +115,14 @@ export default function TokenTable() {
             <TokenDataContainer>
               {tokens.map((token, index) => (
                 <LoadedRow
-                  key={token?.name}
+                  key={token?.address}
                   tokenListIndex={index}
                   tokenListLength={tokens?.length ?? 0}
                   token={token}
-                  ref={tokens.length === index + 1 ? lastTokenRef : undefined}
+                  ref={index + 1 === tokens.length ? lastTokenRef : undefined}
```

Comment:
why this change?

Reply:
lmao because my brain thinks this makes more sense XD 

we are checking if the current index is equal to the tokens length, not vice versa. there is no diff tbh.